dockstore-webservice/src/main/java/io/dockstore/webservice/core/LambdaEvent.java

```diff
@@ -0,0 +1,175 @@
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+/**
+ * This describes events triggered by GitHub webhooks
+ */
+@ApiModel("LambdaEvent")
+@Entity
+@Table(name = "LambdaEvent")
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByRepository", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.repository = :repository"),
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUsername", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.username = :username"),
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUser", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.user = :user"),
+})
+@SuppressWarnings("checkstyle:magicnumber")
+public class LambdaEvent {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @ApiModelProperty(value = "Unique ID of the event.", position = 0)
+    private long id;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The repository from the event.", required = true, position = 1)
+    private String repository;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The name of the user on GitHub that triggers the event.", required = true, position = 2)
+    private String username;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The git reference from the event.", required = true, position = 3)
+    private String reference;
+
+    @Column(nullable = false, columnDefinition = "boolean default true")
+    @ApiModelProperty(value = "Whether or not the event was successful.", position = 4)
+    private boolean success = true;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The message associated with the event.", position = 5)
+    private String message;
+
+    @Column(nullable = false, columnDefinition = "boolean default false")
+    @ApiModelProperty(value = "Whether or not the user has dismissed the event.", position = 6)
+    private boolean dismissed = false;
+
+    @Column
+    @Enumerated(EnumType.STRING)
+    @ApiModelProperty(value = "The type of event.", required = true, position = 7)
+    private LambdaEventType type;
+
+    @ManyToOne
+    @JoinColumn(name = "userId", referencedColumnName = "id")
+    @ApiModelProperty(value = "User that the event is acting on (if exists in Dockstore).", position = 8)
+    @JsonIgnore
+    private User user;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The installation ID of the event.", position = 9)
```

Comment:
Isn't this semi-secret? Probably not, but just double-checking.

Reply:
Not entirely sure, will investigate.

Reply:
After investigating I am not entirely sure, though I think it is ok just to remove it since it is not a requirement. We don't do anything with it, so I am just going to remove it.